rewrite-core/src/main/java/org/openrewrite/internal/lang/NonNull.java

```diff
@@ -40,7 +39,7 @@
 @Target({ElementType.METHOD, ElementType.PARAMETER, ElementType.FIELD, ElementType.TYPE, ElementType.TYPE_USE})
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
-@Nonnull
+@org.jspecify.annotations.NonNull
```

Comment:
I'm seeing this warning when opening up the class in my IDE, so I don't think we can adopt this as is.
![image](https://github.com/user-attachments/assets/a935e337-fb50-4949-adbc-e5eb9c225df2)